src/main/java/com/hivemq/cli/commands/shell/ShellCommand.java

```diff
@@ -243,7 +240,7 @@ static void readFromShell() {
 
 
     static void usage(Object command) {
-        currentCommandLine.usage(command, System.out, MqttCLIMain.COLOR_SCHEME);
+        currentCommandLine.usage(command, System.out, CommandLineProvider.getColorScheme());
```

Comment:
Is this necessary, isn't there a printUsage() method or something similar?

Reply:
This is a wrapper for always printing with the currently active command line in the shell mode. 

There is actually no such method but an alternative would be:
`System.out.println(currentCommandLine.getUsageMessage(CommandLineProvider.getColorScheme()));`

What do you think about this solution?

Reply:
Then it is ok, I just wondered if why you have to provide the color scheme, although it is already set for the currentCommandLine.

Reply:
Actually I think you're right and this gets solved with your proposed Dagger changes. 👍